5.rb

```diff
@@ -0,0 +1,19 @@
+def remove_first_last_two_char(str)
+    result = ""
+    len = str.length
+
+    if len <= 4
+        return result
+        
+    # use pos + .each as .each_with_index
+    pos = 0
+    str.split('').each do 
+        |c|
+        if pos != 0 && pos != 1 && pos != len-1 && pos != len-2
```

Comment:
First 2 and last 2 chars means pos between 2 and ((len - 1) - 2) right?
Please refactor this long if condition